docs/manual/core-concepts/model-querying-basics.md

```diff
@@ -427,14 +427,14 @@ Keep in mind, the provided range value can [define the bound inclusion/exclusion
 
 ```js
 [Op.contains]: 2,            // @> '2'::integer  (PG range contains element operator)
-[Op.contains]: [1, 2],       // @> [1, 2)        (PG range contains range operator)
```

Comment:
These comments are correct, `[1,2)` signifies inclusive/exclusive bounds. Lower bound is inclusive and upper bound is exclusive by default

https://www.postgresql.org/docs/9.5/rangetypes.html

Reply:
Thank you for clarification. I close PR